bugsnag-android-core/src/test/java/com/bugsnag/android/AppDataCollectorSerializationTest.kt

```diff
@@ -31,7 +31,7 @@ internal class AppDataCollectorSerializationTest {
             val memoryTrimState = MemoryTrimState()
 
             // populate summary fields
-            config.appType = "React Native"
+            config.appType = "ReactNative"
```

Comment:
Is this related?

Reply:
This will be fixed once the integration branch is based on the latest from `next`. This is our "all whitespace in json is stripped" friend.